lgc/patch/WaveSizeAdjust.cpp

```diff
@@ -0,0 +1,92 @@
+/*
+ ***********************************************************************************************************************
+ *
+ *  Trade secret of Advanced Micro Devices, Inc.
+ *  Copyright (c) 2019, Advanced Micro Devices, Inc., (unpublished)
+ *
+ *  All rights reserved. This notice is intended as a precaution against inadvertent publication and does not imply
+ *  publication or any waiver of confidentiality. The year included in the foregoing notice is the year of creation of
+ *  the work.
+ *
+ **********************************************************************************************************************/
+/**
+ ***********************************************************************************************************************
+ * @file  WaveSizeAdjust.cpp
```

Comment:
Change the file name. See below comments.

Reply:
Yes, this is a historical problem. We should have a refactoring change to correct all naming. Currently, just follow existing codes.

Reply:
Ok, I will fix the naming issues.

Reply:
Fixed the naming issue. Use prefix Patch before the class name and file name.